segmentstore/storage/src/main/java/io/pravega/segmentstore/storage/chunklayer/ChunkStorageMetrics.java

```diff
@@ -19,6 +20,7 @@
  * Defines all Metrics used by the {@link BaseChunkStorage} class.
  */
 public class ChunkStorageMetrics {
```

Comment:
This class should be declared `static` if all it has is static variables and methods.

Reply:
in java `static` has different meaning than C#.
In C# it mean you can not have instance variable. 

In java you can have nested classes that have reference to _enclosing outer object_. When you have static in java it means don't include ref to outer class. By this definition all top level classes are static already.